dialect/sql/sqlgraph/graph.go

```diff
@@ -347,6 +354,20 @@ func CreateNode(ctx context.Context, drv dialect.Driver, spec *CreateSpec) error
 	return tx.Commit()
 }
 
+// BatchCreate applies the BatchCreateSpec on the graph.
+func BatchCreate(ctx context.Context, drv dialect.Driver, spec *BatchCreateSpec) error {
+	tx, err := drv.Tx(ctx)
+	if err != nil {
+		return err
+	}
+	gr := graph{tx: tx, builder: sql.Dialect(drv.Dialect())}
+	cr := &creator{BatchCreateSpec: spec, graph: gr}
+	if err := cr.nodes(ctx, tx); err != nil {
+		return rollback(tx, err)
+	}
+	return tx.Commit()
```

Comment:
This is a duplicate of `CreateNode` with the exception of how `creator` if initialised. Combining the two using closures.